packages/wallet/test/services/ProviderTracker/TipTracker.test.ts

```diff
@@ -95,6 +100,43 @@ describe('TipTracker', () => {
     });
   });
 
+  it('syncStatus updates emitted only once connection is back up', () => {
+    createTestScheduler().run(({ cold, hot, expectObservable }) => {
+      const syncStatus: Partial<SyncStatus> = { isSettled$: cold('x-ba-|') };
+      const connectionStatusMock$: Observable<ConnectionStatus> = hot('pd--p|', {
+        d: ConnectionStatus.down,
+        p: ConnectionStatus.up
+      });
+      const provider$ = stubObservableProvider<Cardano.Tip>(cold('x|', mockTips), cold('a|', mockTips));
+      const tracker$ = new TipTracker({
+        connectionStatus$: connectionStatusMock$,
+        maxPollInterval: Number.MAX_VALUE,
+        minPollInterval: 0,
+        provider$,
+        store,
+        syncStatus: syncStatus as SyncStatus
+      });
+      expectObservable(tracker$.asObservable()).toBe('x---a', mockTips);
+    });
+  });
+
+  it('syncStatus timeout while connection is down does not emit tip updates', () => {
+    createTestScheduler().run(({ cold, hot, expectObservable }) => {
+      const syncStatus: Partial<SyncStatus> = { isSettled$: hot('10ms |') };
+      const connectionStatusMock$: Observable<ConnectionStatus> = hot('d--|', { d: ConnectionStatus.down });
+      const provider$ = stubObservableProvider<Cardano.Tip>(cold('x|', mockTips));
```

Comment:
Maybe we could also assert that provider$ wasn't subscribed (there's a `expectSubscriptions`)